app/validation/validator.py

```diff
@@ -2099,8 +2135,19 @@ def _get_answers_with_parent_ids(self, json_to_validate):
         answers = {}
         for question, context in self._get_questions_with_context(json_to_validate):
             for answer in question.get("answers", []):
-                answers[answer["id"]] = {"answer": answer, **context}
+
+                if not answer["id"] in answers:
+                    answers[answer["id"]] = {"answer": answer, **context}
+
                 for option in answer.get("options", []):
+                    if answers[answer["id"]]["answer"]["options"] and option[
+                        "value"
+                    ] not in [
+                        option["value"]
+                        for option in answers[answer["id"]]["answer"]["options"]
+                    ]:
+                        answers[answer["id"]]["answer"]["options"].append(option)
+
```

Comment:
As discussed, I'm not sure of the need for this section. I can revert this code and none of the tests fail.

Reply:
So it looks like runner validations will fail without this - there should therefore be a test in place within validator that tests this bit of code.

Reply:
It's a separate method now, thanks! There is invalid schema to test it.